tools/namefixer.py

```diff
@@ -892,10 +896,14 @@
     "play->roomCtx.unk31": "play->roomCtx.status",
 
     "actorCtx.unkC": "actorCtx.halfDaysBit",
-    "actorCtx.unk1F4": "actorCtx.unk_1F4.unk_00",
-    "actorCtx.unk1F5": "actorCtx.unk_1F4.timer",
-    "actorCtx.unk1F8": "actorCtx.unk_1F4.unk_04",
-    "actorCtx.unk1FC": "actorCtx.unk_1F4.unk_08",
+    "actorCtx.unk1F4": "actorCtx.playerImpact.unk_00",
+    "actorCtx.unk1F5": "actorCtx.playerImpact.timer",
+    "actorCtx.unk1F8": "actorCtx.playerImpact.unk_04",
+    "actorCtx.unk1FC": "actorCtx.playerImpact.unk_08",
+    "actorCtx.unk_1F4.unk_00": "actorCtx.playerImpact.unk_00",
+    "actorCtx.unk_1F4.timer": "actorCtx.playerImpact.timer",
+    "actorCtx.unk_1F4.unk_04": "actorCtx.playerImpact.unk_04",
+    "actorCtx.unk_1F4.unk_08": "actorCtx.playerImpact.unk_08",
```

Comment:
Did you forget to put the actual names here? lol

Reply:
lmao